nilearn/plotting/html_connectome.py

```diff
@@ -241,8 +255,45 @@ def view_markers(coords, colors=None, marker_size=5.):
         surface.
 
     """
-    if colors is None:
-        colors = ['black' for i in range(len(coords))]
-    connectome_info = _get_markers(coords, colors)
+    if marker_color is None:
+        marker_color = ['red' for i in range(len(marker_coords))]
+    connectome_info = _get_markers(marker_coords, marker_color)
     connectome_info["marker_size"] = marker_size
     return _make_connectome_html(connectome_info)
+
+
+def _warn_deprecated_params_view_markers(kwargs):
+    """ For view_markers(), raises warnings about deprecated parameters.
+    """
+
+    all_deprecated_params = {'coords': 'marker_coords',
+                             'colors': 'marker_color',
+                             }
+    used_dperecated_params = set(kwargs).intersection(all_deprecated_params)
+    for deprecated_param_ in used_dperecated_params:
+        replacement_param = all_deprecated_params[deprecated_param_]
+        param_deprecation_msg = (
+            'The parameter "{}" will be removed in Nilearn version 0.6.0. '
+            'Please use the parameter "{}" instead.'.format(deprecated_param_,
+                                                            replacement_param,
+                                                            )
+        )
+        warnings.filterwarnings('always', message=param_deprecation_msg)
+        warnings.warn(category=DeprecationWarning,
+                      message=param_deprecation_msg,
+                      stacklevel=3,
+                      )
+
+
+def _transfer_deprecated_param_vals_view_markers(kwargs):
+    """ For view_markers(), reassigns new parameters the values passed
+    to their corresponding deprecated parameters.
+    """
+    coords = kwargs.get('coords', None)
+    colors = kwargs.get('colors', None)
+    
+    if coords is not None:
+        kwargs['marker_coords'] = coords
+    if colors:
```

Comment:
please be explcit `if colors is None` to avoid problems e.g. if colors can be a numpy array

Reply:
You mean `if colors is not None:` right?
Okay, I'll open a new PR and fix this in functions for both the PRs.

Reply:
yes, sorry. in general we need to be wary of implicit conversions to bool, mostly because of numpy arrays